tests/topotests/mgmt_config/r1/frr2.conf

```diff
@@ -0,0 +1,15 @@
+debug northbound notifications
```

Comment:
Can we comment out `debug` statements?

Reply:
I've commented out the northbound debug, the mgmtd debug is what the test is looking at to check for the correct behavior.